GitHubReactions.ps1

```diff
@@ -0,0 +1,363 @@
+
+    $params = @{
+        'UriFragment' = $uriFragment
+        'Description' =  $description
+        'Method' = 'Post'
+        'Body' = @{ content = $ReactionType } | ConvertTo-Json
+        'AcceptHeader' = 'application/vnd.github.squirrel-girl-preview+json'
+        'AccessToken' = $AccessToken
+        'TelemetryEventName' = $MyInvocation.MyCommand.Name
+        'TelemetryProperties' = $telemetryProperties
+        'NoStatus' = (Resolve-ParameterWithDefaultConfigurationValue -Name NoStatus -ConfigValueName DefaultNoStatus)
+    }
+
+    $result = Invoke-GHRestMethod @params
+    if ($PSBoundParameters.ContainsKey('PassThru')) {
+        return $result
+    }
+}
+
+function Remove-GitHubReaction
+{
+<#
+    .SYNOPSIS
+        Removes a reation on a given GitHub issue.
+
+    .DESCRIPTION
+        Removes a reation on a given GitHub issue.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Issue
+        The issue number.
+
+    .PARAMETER ReactionId
+        The Id of the reaction. You can get this from using Get-GitHubReaction.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Remove-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionId 1234
+
+        Remove a reaction by Id on Issue 12626 from the PowerShell\PowerShell project.
+
+    .EXAMPLE
+        Get-GitHubReaction -OwnerName PowerShell -RepositoryName PowerShell -Issue 12626 -ReactionType rocket | Remove-GitHubReaction
+
+        Gets a reaction using Get-GitHubReaction and pipes it into Remove-GitHubReaction.
+
+    .NOTES
+        Currently, this only supports reacting to issues. Issue comments, commit comments and PR comments will come later.
+#>
+    [CmdletBinding(
+        SupportsShouldProcess,
+        DefaultParameterSetName='Elements')]
```

Comment:
With #171 now resolved, moving forward, I think all `Remove-*` functions should have confirmation, along with an additional CBH example explaining how to avoid confirmation.  Review #174 to see what would need to be added to make that work here.

Reply:
Fixed... Although, I'd recommend a `-Force` to skip over confirmation since `-Confirm:$false` is tedious.

Here's a really awesome article about this:
https://powershellexplained.com/2020-03-15-Powershell-shouldprocess-whatif-confirm-shouldcontinue-everything/#implementing--force

Reply:
Opened an issue about that:
https://github.com/microsoft/PowerShellForGitHub/issues/218